cyences_app_for_splunk/default/macros.conf

```diff
@@ -12,7 +12,11 @@ iseval = 0
 
 [cs_filter_savedsearches(1)]
 args = name
-definition = regex savedsearch_name="($name$)\s" | rex field=savedsearch_name "^.*\-\s(?<Name>.*)$"
+definition = eval products_to_search="$name$" | makemv delim="|" products_to_search | eval products_to_search=trim(products_to_search) \
+| cyencesmvintersection set1="products_to_search" set2="alert_products" \
```

Comment:
Just to make sure values are not passed as a string to set1 and set2

Reply:
They are field names passed to set1 and set2

That's how we are doing for mvdiff command. Not sure if there is a better way to do it.